src/InvokePlaster.ps1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
Ah, I did change the encoding of that file, as it was UTF-8, which generates the PSSA warning `PSUseBOMForUnicodeEncodedFile`. I can revert that if it causes an issue, but I couldn't find one in my testing.